.travis.yml

```diff
@@ -7,11 +7,6 @@ os: osx
 osx_image: beta-xcode6.1
 
 env:
-  matrix:
-    
-    - CONDA_PY=27
-    - CONDA_PY=34
-    - CONDA_PY=35
```

Comment:
Actually just reopened issue ( https://github.com/conda-forge/conda-smithy/issues/223 ) instead. Please just revert for now.